py3status/modules/yubikey.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+"""
+Show an indicator when YubiKey is waiting for a touch.
+
+Configuration parameters:
+    format: Display format for the module.
+        (default '\?if=is_waiting YubiKey')
+    socket_path: A path to the yubikey-touch-detector socket file.
+        (default '$XDG_RUNTIME_DIR/yubikey-touch-detector.socket')
+
+Control placeholders:
+    {is_waiting} a boolean, True if YubiKey is waiting for a touch for any reason.
+    {is_waiting_gpg} a boolean, True if YubiKey is waiting for a touch due to a gpg operation.
+    {is_waiting_u2f} a boolean, True if YubiKey is waiting for a touch due to a pam-u2f request.
```

Comment:
you mean to remove the `is_waiting` and have the default format be something like this?

```
format = '\?if=is_waiting_gpg OR is_waiting_u2f'
```

what's the correct syntax for this?

Reply:
`format = '[\?if=is_gpg KEY][\?is_u2f KEY]'`

Reply:
@lasers just stumbled upon an issue with this format, it is possible that there are two commands issued for Yubikey at the same time (e.g. pacman is updating packages and requesting sudo via u2f, while at the same time on a different tab you ssh somewhere and this triggers gpg check). This results in module output "YubiKeyYubiKey", because `is_gpg` and `is_u2f` are both active at the same time. I want a true OR, not two separate ifs - how can we achieve that?

Reply:
I need this for a case when I don't want to see `GPG` or `U2F`, I just want to see `YubiKey` (or in my case, I use a symbol of key).

But using your example I think I solved it, this seems to work like a true OR:

```
format = '[[\?if=is_gpg ][\?if=is_u2f ] Yubikey]'
```

Reply:
Nice. `format = '[YubiKey[\?if=is_gpg ][\?if=is_u2f ]]'` seems to work too.

Reply:
Indeed, I'll use your way as a default, it's easy to understand how to go from it to additionally showing `GPG` or `U2F`.